### PR TITLE
Ensure pull-release-verify securityContext is on container

### PR DESCRIPTION
**Pull request by spiffxp**
ref: https://github.com/kubernetes/test-infra/pull/13258
ref: https://github.com/kubernetes/test-infra/pull/13125

### CODE REVIEW

File: config/jobs/kubernetes/release/release-config.yaml [hunk 1]
```diff
@@ -61,5 +61,5 @@ presubmits:
           args:
             - make
             - verify
-      securityContext:
-        privileged: true
+        securityContext:
+          privileged: true
```
Comment (spiffxp) [inline config/jobs/kubernetes/release/release-config.yaml]: @stevekuznetsov @cjwagner could we have caught this via a presubmit?

**Comment by spiffxp** [comment]
🔥 YAML 🔥

**Comment by spiffxp** [comment]
FYI @pswica

**Comment by spiffxp** [comment]
Should address https://github.com/kubernetes/release/issues/757

**Comment by pswica** [comment]
> 🔥 YAML 🔥

Hahaha. I was actually on vim trying to trace my lines doing this too...

**Pull request merged**